templates/web/base/dashboard/index.html

```diff
@@ -24,114 +30,30 @@
-    </tr>
-    [% END %]
-
-    <tr class='subtotal' id="total_fixed">
-        <th scope="row">[% loc('Total marked as fixed') %]</th>
-        <td>[% problems.wtd.${"fixed - council"} + problems.wtd.fixed_user %]</td>
-        <td>[% problems.week.${"fixed - council"} + problems.week.fixed_user %]</td>
-        <td>[% problems.weeks.${"fixed - council"} + problems.weeks.fixed_user %]</td>
-        <td>[% problems.ytd.${"fixed - council"} + problems.ytd.fixed_user %]</td>
-    </tr>
-
-    [%
-    rows = {
-        '0' => [ "in progress", loc("Council has marked as in progress") ]
-        '1' => [ "action scheduled", loc("Council has marked as planned") ]
-        '2' => [ "investigating", loc("Council has marked as investigating") ]
-        '3' => [ "closed", loc("Council has marked as closed") ]
-    };
-    wtd = 0, week = 0, weeks = 0, ytd = 0;
-    FOR row IN rows %]
-    <tr id="[% row.value.0.replace('[^\w]+', '_' ) %]">
-        <th scope="row">[% row.value.1 %]</th>
-        <td>[% problems.wtd.${row.value.0} %]</td>
-        <td>[% problems.week.${row.value.0} %]</td>
-        <td>[% problems.weeks.${row.value.0} %]</td>
-        <td>[% problems.ytd.${row.value.0} %]</td>
-    </tr>
-    [% END %]
-
-    <tr class='subtotal' id="marked">
-        <th scope="row">[% loc('Total marked') %]</th>
-        <td>[% problems.wtd.${"in progress"} + problems.wtd.${"action scheduled"} +
-            problems.wtd.investigating + problems.wtd.closed %]</td>
-        <td>[% problems.week.${"in progress"} + problems.week.${"action scheduled"} +
-            problems.week.investigating + problems.wtd.closed %]</td>
-        <td>[% problems.weeks.${"in progress"} + problems.weeks.${"action scheduled"} +
-            problems.weeks.investigating + problems.wtd.closed %]</td>
-        <td>[% problems.ytd.${"in progress"} + problems.ytd.${"action scheduled"} +
-            problems.ytd.investigating + problems.wtd.closed %]</td>
-    </tr>
-
-    <tr id="avg_fixed">
-        <th scope="row">[% loc('Average time to council marking fixed (days)') %]</th>
-        <td>[% problems.wtd.time_to_fix %]</td>
-        <td>[% problems.week.time_to_fix %]</td>
-        <td>[% problems.weeks.time_to_fix %]</td>
-        <td>[% problems.ytd.time_to_fix %]</td>
-    </tr>
+  [% ELSE %]
 
-    <tr id="avg_marked">
-        <th scope="row">[% loc('Average time to first council state change (days)') %]</th>
-        <td>[% problems.wtd.time_to_mark %]</td>
-        <td>[% problems.week.time_to_mark %]</td>
-        <td>[% problems.weeks.time_to_mark %]</td>
-        <td>[% problems.ytd.time_to_mark %]</td>
-    </tr>
-
-    <tr class='subtotal' id="not_marked">
-        <th scope="row">[% loc('Total not marked') %]</th>
-        <td>[% problems.wtd.not_marked %]</td>
-        <td>[% problems.week.not_marked %]</td>
-        <td>[% problems.weeks.not_marked %]</td>
-        <td>[% problems.ytd.not_marked %]</td>
-    </tr>
-
-</table>
```

Comment:
Are we removing all these stats on time to state change?

Reply:
Yes. Not sure if anyone using them, plus they haven't been correct for ages (ignoring most closed states, for example). The marketing page still has an average, and it can always be put back somewhere if missed, but I thought it best to cut back as much as possible first.